tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/signup_info.h

```diff
@@ -59,10 +57,13 @@ SignupInfo* deserialize_signup_info(
     const std::string& serialized_signup_info);
 
 std::map<std::string, std::string> CreateEnclaveData(
-    const std::string& originator_public_key_hash);
+    uint64_t in_mode,
+    const std::string& in_ext_data,
+    const std::string& in_ext_data_signature,
+    const std::string& in_kme_attestaion);
```

Comment:
Change in_kme_attestaion to in_kme_attestation

Reply:
Done